pandunia-lekse.csv

```diff
@@ -3253,6 +3310,7 @@ xowe|||eng:deu:fra:spa:por:show, rus:шоу (šou), tur:şov, swa:shoo, tha:โ
 xudu|||eng:should|should (ought)||deber (se recomienda)|||||||||||||devus|pitäisi|powinien
 xukare|||ara:(sukkar), swa:sukari, por:açúcar, spa:azúcar, hin:शक्कर (śakkar), ben:শর্করা (śôrkôra), tur:şeker, tam:சக்கரை (cakkarai)|sugar|sucre|azúcar|açúcar|сахар|سُكَّر|糖|砂糖|||शक्कर|চিনি (শর্করা)|sakar|sukari|şeker|sukero|sokeri|cukier
 xukari||||sugary (sweet)||azucarado||||||||||||||sokerinen (makea)|słodki
+xukari bete||||sugar beet|betterave à sucre (betterave sucrière, betterave)|remolacha azucarera||сахарная свёкла|||テンサイ (サトウダイコン)||||||||||
```

Comment:
Why not "xukarbete"?

Reply:
This was just based on the convention I made up where families and genuses get compounds and roots, while individual species and subspecies just get adjectives to modify the genus name. I'm not wedded to it; this can be xukarbete instead. I would just also change "baki margarite" to "bakomargarite", etc to keep it all consistent.

Reply:
I see. I think that it's best that each name is a word. Then everybody can recognize that it's a thing, not a phrase.
That's my opinion in principle. I don't know is it possible to carry it out always in practice.

Reply:
Let's try the "xukarbete" way for now, and if it turns out to be impossible in practice then we can reevaluate. I noticed many adjective-noun phrases in the dictionary are becoming compounds, so this will be more consistent with the rest of the language.